forge/test/mlir/test_ops.py

```diff
@@ -14,6 +14,30 @@
 from forge.tensor import to_forge_tensors, to_pt_tensors
 
 
+@pytest.mark.parametrize(
+    "num_channels, height, width",
+    [
+        (32, 56, 56),
+        (64, 112, 112),
+        (128, 224, 224),
+        (256, 28, 28),
+    ],
+)
+def test_multidim_unsqueeze(num_channels, height, width):
+
+    framework_model = nn.BatchNorm2d(num_features=num_channels)
+
+    inputs = [torch.randn(1, num_channels, height, width)]
```

Comment:
For batch norm, can we also test multiple batches?

Reply:
- Observed pcc drop when Batch size >1.

| Input Shape         | PCC              |
|---------------------|------------------------|
| (2, 64, 112, 112)   | 0.6199620538910243      |
| (4, 64, 28, 28)     | 0.4935656199688308      |
| (8, 64, 112, 112)   | 0.40443518583193394     |
| (16, 128, 224, 224) | -0.0004391043640747615  |
| (32, 256, 28, 28)   | 0.39200606381500713     |

Reply:
Make sense, let's focus on batch 1, but please leave a comment in the test these details regarding PCC.